### PR TITLE
#51 implemented snooze feature

### PR DESCRIPTION
**Pull request by Rajendra-R**
connected to #51

### CODE REVIEW

File: gitkanban/gitkanban.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 PEOPLES_BLACKLIST = ["deepcompute-agent", "deep-compute-ops"]
 DEFAULT_TIME_ELAPSED = "2h"
 MAX_RETRIES = 3
+INGPROGRESS_LABEL = 'inprogress-status'
```
Comment (prashanthellina) [inline gitkanban/gitkanban.py]: @Rajendra-R We shouldn't be hardcoding this. Should come from Config file.

File: gitkanban/gitkanban.py [hunk 2]
```diff
@@ -45,6 +46,17 @@
     "system-owner"
 ]
 
+SNOOZE_LABELS = [
```
Comment (prashanthellina) [inline gitkanban/gitkanban.py]: @Rajendra-R We shouldn't be hardcoding this. Should come from Config file.

File: gitkanban/gitkanban.py (hunk 1 above)
Comment (Rajendra-R) [reply gitkanban/gitkanban.py]: Sure will do it.

File: gitkanban/gitkanban.py (hunk 2 above)
Comment (Rajendra-R) [reply gitkanban/gitkanban.py]: Sure will do it.

**Pull request merged**